Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -297,11 +297,16 @@ CGameListCtrl::~CGameListCtrl()
 
 template <typename T>
 static void InitBitmap(wxImageList* img_list, std::vector<int>* vector, wxWindow* context,
-                       const wxSize& usable_size, T index, const std::string& name)
+                       const wxSize& usable_size, T index, const std::string& name,
+                       bool themed = false)
 {
   wxSize size = img_list->GetSize();
-  (*vector)[static_cast<size_t>(index)] = img_list->Add(WxUtils::LoadScaledResourceBitmap(
-      name, context, size, usable_size, WxUtils::LSI_SCALE | WxUtils::LSI_ALIGN_VCENTER));
+  auto bitmap =
+      themed ? WxUtils::LoadScaledThemeBitmap(name, context, size, usable_size,
```

Comment:
Since these functions have the same call signature, you could deduplicate this to something like:

```c++
auto func = themed ? WxUtils::LoadScaledThemeBitmap :
                     WxUtils::LoadScaledResourceBitmap;
func(name, context, size, usable_size,
     WxUtils::LSI_SCALE | WxUtils::LSI_ALIGN_VCENTER);
```